src/client/components/AutoSearch.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import { Link } from "react-router-dom";
+
+const AutoSearch = () => {
+    const [data, setData]=useState([])
```

Comment:
This is my personal trigger, everything in software is data! 
What is special about this data? Put that into the name. It seems like searchResults?